include/qpdf/QPDFObjectHandle.hh

```diff
@@ -63,7 +63,7 @@ class Pl_QPDFTokenizer;
 class QPDFMatrix;
 class QPDFParser;
 
-class QPDFObjectHandle final: public qpdf::BaseHandle
```

Comment:
No object, but I'm just curious as to why. I don't have a strong opinion about whether it should or shouldn't be final.